docs/quickstarts/README.md

```diff
@@ -115,6 +115,40 @@ If you have created quick starts in the Hybrid Cloud Console before, start here:
 
     e. Get your quick start reviewed by stakeholders as needed.
 
+6. On your issue branch, add new or changed quick start files to stage.
+
+    ```
+     $[issue-branch] git add -A 
+     
+    ```
+    (the espanso shortcut is `:add`)
```

Comment:
Let's delete the reference to espanso, as it's not a tool everyone uses and these docs are public.

Reply:
@dayleparker Ok, that's fine. Got stuck on the Insights in the name of the repos. :-)